receiver/awslambdareceiver/Makefile

```diff
@@ -1 +1,10 @@
-include ../../Makefile.Common
\ No newline at end of file
+include ../../Makefile.Common
+
+.PHONY: install-deps
+install-deps:
+	go install go.uber.org/mock/mockgen@latest
+
+.PHONY: mockgen
+mockgen:
+	mockgen -source=./internal/s3.go -destination=./internal/mock_s3_service.go -package=internal
+	mockgen -source=./internal/custom_trigger.go -destination=./internal/mock_custom_trigger.go -package=internal
```

Comment:
Rather than installing it, how about adding a tool directive to go.mod? https://go.dev/ref/mod#go-mod-file-tool

Then you can use `go tool go.uber.org/mock/mockgen` below